source/funkin/modding/ModStore.hx

```diff
@@ -0,0 +1,59 @@
+package funkin.modding;
+
+import haxe.ds.StringMap;
+
+/**
+ * Temporary persistent data storage for mods to use.
+ */
+class ModStore
+{
+  /**
+   * All registered stores for this runtime.
+   */
+  public static final stores:StringMap<Dynamic> = new StringMap<Dynamic>();
```

Comment:
Is this supposed to be a final? Considering you're modifying it.

Reply:
the final just means that we can't reassign the stores map.
we can still modify the values inside of the stores map

Reply:
Ah, my bad.